pkg/controller/bootstrap/bootstrap.go

```diff
@@ -89,6 +89,7 @@ func (b *Bootstrap) Run(destDir string) error {
 		idmsRules            []*apicfgv1.ImageDigestMirrorSet
 		itmsRules            []*apicfgv1.ImageTagMirrorSet
 		clusterImagePolicies []*apicfgv1alpha1.ClusterImagePolicy
+		imagePolicies        []*apicfgv1alpha1.ImagePolicy
```

Comment:
@wking @mtrmac I’m unsure about adding this to bootstrap. 
I tested it by adding a  manifest to the manifest directory before manually creating the cluster, and the object only appears when I set the metadate.namespace to `openshift-machine-config-operator`. 
I didn’t see the object when using `openshift-config` or `openshift-cluster-version`(I only tried these namespaces). In the enhancement, I noticed in the [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/security/openshift-image-policy.md#use-an-imagepolicy-for-openshift--namespaces) we might want to use ImagePolicy for release payload verifications—could this be an issue in the future?

Reply:
ah, yeah, I'd been hoping to eventually have an `openshift-cluster-version` ImagePolicy set up by the CVO by default.  I'm agnostic about whether that needs to be bootstrap-time or not, but [OCPBUGS-37770](https://issues.redhat.com/browse/OCPBUGS-37770) had installs failing when the production CVO wanted a policy that wasn't in place at bootstrap-time.